lib/Optimizer/Lower/Lower.cpp

```diff
@@ -1906,7 +1955,8 @@ bool glow::lowerNode(Function *F, Node *node, CompilationContext &cctx) {
     CASE_LOWER(LSTMUnit);
     CASE_LOWER(Broadcast);
     CASE_LOWER(LogSoftMax);
-    CASE_LOWER(HardSwish);
+    CASE_DEQUANT(Tanh, ElemKind::FloatTy);
+    CASE_LOWER_QUANT(HardSwish, ElemKind::FloatTy);
```

Comment:
We have hard swish lowering implemented at this point in master. Can you rebase?